src/autora/doc/pipelines/main.py

```diff
@@ -15,6 +18,27 @@
     format="%(asctime)s %(levelname)s %(module)s.%(funcName)s(): %(message)s",
 )
 logger = logging.getLogger(__name__)
+nltk.download("wordnet")
```

Comment:
Can you move this code to it's own module? E.g. `metrics.py`. Then we can import it not at the top of main, but inside the `eval()` method, like `mlflow` and `jsonlines`. That way wordnet is only downloaded when computing metrics.

Reply:
Done!